src/lib/pipeline.ml

```diff
@@ -324,9 +328,11 @@ module Construct = struct
        make_target }
       bam_pair
 
-end
+  let rna_hla_typer fastq_sample = Rna_hla_typer fastq_sample
```

Comment:
> The code is in Pipeline because of the non-extensibility we discussed yesterday (Vs circular dependencies).

I don't think there is a necessary circular dependence. We can solve this problem the way that it is _always_ solved in OCaml. We can split `Pipeline` into a part where we declare the types, use those types in tool specific modules and then recombine them into a module containing shallow constructors ala `Pipeline.Specific` above.

> If a user starts using Construct.rna_hla_typer it means nothing, and looks like the "default" HLA typer may change in the future. Biokepi authors should not choose which tool is best/default for a task, that's up to the user (who most likely knows more biology than us, and what they actually want).

Agreed, but again, at this moment there is no support for another hla_typer (just like there is no support for another `gzip`). Why solve this problem until it is necessary. Feel free to file an issue if you think that we would forget.